pagopa/io-dev-api-server
# Debug configuration for VSCode

## Short description
This PR adds the VSCode configuration to launch the `io-dev-api-server` in debug mode and use the debug capabilites (breakpoints, variables, call stack, ...)

![Screenshot 2023-01-13 at 18 02 01](https://user-images.githubusercontent.com/5150343/212377382-f1d60da1-8585-4365-9f54-9b9fa65b9c7a.png)

## List of changes proposed in this pull request
- Added `.vscode/launch.json` configuration
- Added the previous file to git and excluded from the `.gitignore` file
- Node version bumped to 16.19.0

## How to test
Open the project from VSCode, from the Run menu, select Start Debugging and check that the server starts in debug with the appropriate VSCode view. 
Place any breakpoint that you want (maybe in some HTTP request handler), start a request and notice that the breakpoint is hit.
